# Apigo-and-Oono-2018

11.	Appendix  
Supplementary material and reproducible code for bioinformatic processing, sequence removal and host specificity measurements can be accessed on Github at https://github.com/austenapigo.

Appendix A. Bioinformatic pipeline using USEARCH (version 9.2.64; Edgar 2010), BLAST+ (Camacho et al. 2008), MEGAN (Huson et al. 2007) and QIIME (Caporaso et al. 2010).

Appendix B. OTU tables used for structural, network and phylogenetic specificity with fungal taxonomic annotations.

Appendix C. R code for (1) sequence removal using Multivariate Cutoff Level Analysis (MultiCoLA; Gobet et al. 2010), (2) structural (vegan; Oksanen et al. 2017), network (bipartite; Dormann et al. 2008; Dormann 2011) and phylogenetic (picante; Kembel et al. 2010) specificity, and (3) randomized community analysis using the ‘nullmodel’ function (bipartite; Dormann et al. 2008) for structural and network specificity and ‘ses.mpd’ function (picante; Kembel et al. 2010) for phylogenetic specificity.

Appendix D. Host species phylogenetic tree used to calculate phylogenetic specificity in Newick format. 

Appendix E. Table of p-values for Figures 2, 3 and 6.
